### PR TITLE
modify style login and register page

### PR DESCRIPTION
**Pull request by Ambradibi**
Redesign login page
Remove “Need Support” from login page

### CODE REVIEW

File: aegis-web/yo/app/styles/style.css [hunk 1]
```diff
@@ -85,9 +85,12 @@ a:focus {
...
   background-color: #fafafa;
-  border: none;
+  border-bottom: 1px solid #2e353d;
+  border-bottom-style: double;
+  border-width: medium;
+  border-left: none;
+  border-right: none;
   border-radius: 0;
-  box-shadow: 0 10px 30px rgba(0, 0, 0, 0.25);
 }
```
Comment (cbotsikas) [inline aegis-web/yo/app/styles/style.css:101]: You shouldn't change this class, it's used in many pages

**Review by cbotsikas — CHANGES_REQUESTED** [review]
I don't know how and why, but you are reverting required changes made in style.css for extended metadata ([commit 1](https://github.com/aegisbigdata/hopsworks/commit/9cc62b7c63983d23b3200f7b56d8ed08d77e840c#diff-4fac88e88c06c24a0560f980745c2446), [commit 2](https://github.com/aegisbigdata/hopsworks/commit/1a4dbf3e6c4475621cacae579b9b8330b8ce6355#diff-4fac88e88c06c24a0560f980745c2446)).
You need to fix this so that we only keep your required changes. In order to do so, from your `login-and-register-page` branch you can fix the file and then commit or more preferably, amend the commit. You can have a look on [Option 1 and 2 of this article](https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request)
Also note that you shouldn't change `.panel` style, it's used all over the platform.

File: aegis-web/yo/app/styles/style.css (hunk 1 above)
Comment (Ambradibi) [reply aegis-web/yo/app/styles/style.css:101]: ok. I don't know. I followed the instructions: pull from dev, add, commit, push. Commit 1 and 2 are on dev? I haven't modified its parts.

Comment (Ambradibi) [reply aegis-web/yo/app/styles/style.css:101]: I've add an other class for only the login panel

Comment (cbotsikas) [reply aegis-web/yo/app/styles/style.css:101]: yes those commits are already merged in dev, 17 days ago. Your dev branch also has them. Did you overwrite style.css with an older file you had prepared?

Comment (Ambradibi) [reply aegis-web/yo/app/styles/style.css:101]: no, I didn't. I take his file and I modify it

Comment (Ambradibi) [reply aegis-web/yo/app/styles/style.css:101]: is it ok now?

**Review by cbotsikas — APPROVED** [review]

**Review by cbotsikas — CHANGES_REQUESTED** [review]
Login form is broken. You shouldn't move the submit button outside form tag.

**Comment by Ambradibi** [comment]
I've already solved. Now it's working

**Comment by cbotsikas** [comment]
> I've already solved. Now it's working

you need to push the changes.

**Comment by Ambradibi** [comment]
> > I've already solved. Now it's working
> 
> you need to push the changes.

Done

**Comment by cbotsikas** [comment]
I don't see any changes.

**Comment by cbotsikas** [comment]
Also, based on your commit's timestamps, you need to fix your computer's time. it's set 7 days back

**Comment by Ambradibi** [comment]
> Also, based on your commit's timestamps, you need to fix your computer's time. it's set 7 days back

now yes. 
yes I know, but it does not keep it in memory and I do not know why.

**Comment by Ambradibi** [comment]
every time the machine is restarted. now I have fix.

**Review by cbotsikas — APPROVED** [review]

**Pull request merged**